ansible_collections/arista/avd/plugins/action/verify_requirements.py

```diff
@@ -100,6 +113,10 @@ def _validate_python_requirements(requirements: list, info: dict) -> bool:
             msg = f"Wrong format for requirement {raw_req}"
             raise AnsibleActionFail(msg) from exc
 
+        if RUNNING_FROM_SOURCE and req.name == "pyavd":
```

Comment:
@ClausHolbechArista this is another change I have made when running from source - as when I inject the path I end up with three locations for pyavd if someone has added an editable install beforehand